### DISCUSSION TITLE
unable to verify your academic affiliation

### DISCUSSION THREAD

**Discussion by labanidas** [community/community#48492] [GitHub Education]
I have applied for Student Developer Pack 8 times and all the time my request got rejected. I have been uploading my college Id card and recent fee receipt and each time I received an email like this:

`Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Feb 24, 2023 for @labanidas

The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.

You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

Once you have resolved the issues with your sumbission, [visit us again and reapply](https://education.github.com/). `

Is there something I am missing ?

This [blog ](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/) only for reference

**Comment by dheerazsah** [comment]
This might have been occurred because of the variation of your actual location and the address of your college/ university. If your university is way far than your actual location as you're studying in an affiliated college then I will suggest you to mention your college ID not from your university but from your local college which is the affiliated one.

**Comment by samus-aran** [comment]
Hi @labanidas 👋 

Here are some tips on applying to Global Campus verification process:

- Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.
- For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.

Hope this helps. Let us know how you get on 👍
